training/mozilla_voice_stt_training/train.py

```diff
@@ -636,17 +636,23 @@ def __call__(self, progress, data, **kwargs):
                         break
 
                     # Reduce learning rate on plateau
-                    if (FLAGS.reduce_lr_on_plateau and
-                            epochs_without_improvement % FLAGS.plateau_epochs == 0 and epochs_without_improvement > 0):
-                        # If the learning rate was reduced and there is still no improvement
-                        # wait FLAGS.plateau_epochs before the learning rate is reduced again
+                    # If the learning rate was reduced and there is still no improvement
+                    # wait FLAGS.plateau_epochs before the learning rate is reduced again
+                    if (FLAGS.reduce_lr_on_plateau and epochs_without_improvement > 0
```

Comment:
there is no change of logic here, please avoid changing ordering

Reply:
You're right, this should improve readability but it's better to split this to an extra pr.